### PR TITLE
MGMT-4526 - Extended Event API

### PR DESCRIPTION
**Pull request by slaviered**
Add JSON-formatter properties field to Event to hold free-formatted
    additional properties and optional extra properties parameters to
    Event API for hosting metrics parameters

    additional internal severity filters out internal events from the UI

### CODE REVIEW

File: internal/events/event.go [hunk 1]
```diff
@@ -21,7 +22,7 @@ type Handler interface {
 	//     host added to cluster, we have the host-id as the main entityID and
 	//     the cluster-id as another ID that this event should be related to
 	// otherEntities arguments provides for specifying mor IDs that are relevant for this event
-	AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time)
+	AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{})
```
Comment (gamli75) [inline internal/events/event.go:25]: can you please add a comment with prop examples and also some explanation about this field? something similar to what you have in your testing

Comment (slaviered) [reply internal/events/event.go:25]: I will when I'll added the actual implementation, so to make sure it is accurate

**Review by gamli75 — APPROVED** [review]

**Review by gamli75 — APPROVED** [review]

**Pull request merged**